etc/taskcluster/decision_task.py

```diff
@@ -342,7 +342,7 @@ def linux_wpt():
     def linux_run_task(name):
         return linux_task(name).with_dockerfile(dockerfile_path("run"))
     wpt_chunks("Linux x64", linux_run_task, release_build_task, repo_dir="/repo",
-               total_chunks=2, processes=24)
+               total_chunks=8, processes=4)
```

Comment:
With reduced parallelism, maybe we can get away with cheaper instance types? https://tools.taskcluster.net/aws-provisioner/servo-docker-worker/view is currently configured to use `c5.4xlarge` with `capacity: 1` (number of tasks that can run at the same time on one worker), or `c5.9xlarge` with `capacity: 2`. (Currently these instances are billed to the Taskcluster/Firefox account on AWS, since there’s only one `aws-provisioner`, so maybe they’re a rounding error compared to the cost of Firefox CI.)